src/preprocess.rkt

```diff
@@ -9,16 +9,24 @@
 
 ;; See https://pavpanchekha.com/blog/symmetric-expressions.html
 (define (find-preprocessing initial specification context)
+  ;; f(x) = f(-x)
   (define even-identities
     (reap [sow]
       (for ([variable (in-list (context-vars context))]
             [representation (in-list (context-var-reprs context))])
-        (with-handlers ([exn:fail:user:herbie? (const (void))])
-          ; TODO: Handle case where neg isn't supported for this representation
+        (with-handlers ([exn:fail:user:herbie? (const #f)])
           (define negate (get-parametric-operator 'neg representation))
           ; Check if representation has an fabs operator
           (define fabs (get-parametric-operator 'fabs representation))
           (sow (replace-vars (list (cons variable (list negate variable))) specification))))))
+  ;; f(x) = -f(-x)
+  (define odd-identities
+    (with-handlers ([exn:fail:user:herbie? (const empty)])
+      (define negate (get-parametric-operator 'neg (context-repr context)))
+      ; Check if representation has an fabs operator
+      (define fabs (get-parametric-operator 'fabs (context-repr context)))
+      (map (lambda (expression) (list negate expression)) even-identities)))
```

Comment:
I think in principle this isn't necessarily the right `neg` operator if one of the variables has a different repr than the expression as a whole, but that doesn't ever actually happen in Herbie at the moment, so not sure if this is really worth fixing. The parametric-operator thing is a bit broken anyway.